ase/clapplugin.cc

```diff
@@ -398,10 +406,39 @@ class ClapAudioWrapper : public AudioProcessor {
     output_events_.resize (0);
   }
   void
+  update_transportinfo()
+  {
+    const auto& trans = transport();
+    const auto& tick_sig = trans.tick_sig;
+    transportinfo.flags = CLAP_TRANSPORT_HAS_TEMPO |
+                          CLAP_TRANSPORT_HAS_BEATS_TIMELINE |
+                          CLAP_TRANSPORT_HAS_SECONDS_TIMELINE |
+                          CLAP_TRANSPORT_HAS_TIME_SIGNATURE;
+    transportinfo.tempo      = tick_sig.bpm();
+    transportinfo.tsig_num   = tick_sig.beats_per_bar();
+    transportinfo.tsig_denom = tick_sig.beat_unit();
+    double beat_pos = 0, sec_pos = 0;
+    if (trans.running())
+      {
+        transportinfo.flags |= CLAP_TRANSPORT_IS_PLAYING;
+        beat_pos = trans.current_tick_d * (1.0 / TRANSPORT_PPQN);
+        sec_pos  = trans.current_seconds + trans.current_minutes * 60;
+      }
+    transportinfo.song_pos_beats = llrint (beat_pos * CLAP_BEATTIME_FACTOR);
+    transportinfo.song_pos_seconds = llrint (sec_pos * CLAP_SECTIME_FACTOR);
+    if (0)
+      {
+        printf ("bpm = %f signature=%d/%d beat_pos = %f seconds = %f running=%d\n",
```

Comment:
The Ase namespace has printout() and printerr() if something needs to be printed to stdout/stderr. (Those are typesafe and avoid segfaults).
But since this code is disabled anyway, please submit a new patch set (PR force push would work) with debugging code removed.

Reply:
I removed the debugging code, rebased the branch and force pushed.